pkg/networkmanager/interfaces.js

```diff
@@ -1647,12 +1677,14 @@ PageNetworking.prototype = {
     },
 
     enter: function () {
+        var self = this;
+
         this.log_box = journal.logbox([ "_SYSTEMD_UNIT=NetworkManager.service",
                                        "_SYSTEMD_UNIT=firewalld.service" ], 10);
         $('#networking-log').empty().append(this.log_box);
 
         $(this.model).on('changed.networking', $.proxy(this, "update_devices"));
-        this.update_devices();
+        self.update_devices();
```

Comment:
Good catch.  This is a residual from changing the code, and then throwing away the changes.